flow/layers/opacity_layer.cc

```diff
@@ -87,8 +79,7 @@ void OpacityLayer::Paint(PaintContext& context) const {
   // Skia may clip the content with saveLayerBounds (although it's not a
   // guaranteed clip). So we have to provide a big enough saveLayerBounds. To do
   // so, we first remove the offset from paint bounds since it's already in the
-  // matrix. Then we round out the bounds because of our
-  // RasterCache::GetIntegralTransCTM optimization.
+  // matrix. Then we round out the bounds.
```

Comment:
Perhaps we introduced the offset into the CTM prematurely above? If we hadn't already added it, we wouldn't have to adjust for it here...?

Reply:
(aka line 70 in this method)